src/components/MarketMySharesForm/index.js

```diff
@@ -316,7 +316,7 @@ class MarketMySharesForm extends Component {
                 <label>New predicted value</label>
                 <span>
                   <DecimalValue value={newScalarPredictedValue} />&nbsp;
-                  <CurrencyName collateralToken={market.event.collateralToken} />
+                  <CurrencyName outcomeToken={market.eventDescription.unit} />
```

Comment:
The scalar outcome unit is not the currency, this component is used for displaying ERC20 tokens like the "Short" token or the "Long" token. Please just output `{market.eventDescription.unit}` here. In the future we can pass the contract address here and the "CurrencyName" component will resolve the name